fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ResourceManagerHelperImpl.java

```diff
@@ -160,15 +170,21 @@ public static List<ResourceReloader> sort(ResourceType type, List<ResourceReload
 	}
 
 	protected void sort(List<ResourceReloader> listeners) {
-		listeners.removeAll(addedListeners);
+		listeners.removeAll(listenerFactories);
```

Comment:
This doesn't seem right since the types don't match (`Collection.removeAll` has an unhelpful `Collection<?>` param)

Reply:
Oh yeah, great spot! thanks. I will fix tomorrow.